linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_disk_error_action/ansible/shared.yml

```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- (xccdf-var var_auditd_disk_error_action)
+
+- name: Configure auditd Disk Error Action on Disk Error
+  lineinfile:
+    dest: /etc/audit/auditd.conf
+    line: "disk_error_action = {{ var_auditd_disk_error_action }}"
+    state: present
+  #notify: reload auditd
+  tags:
+    @ANSIBLE_TAGS@
```

Comment:
This Ansible snippet doesn't have a `regexp` expression.
If `/etc/audit/auditd.conf ` contains a line setting `disk_error_action` to a different value than set in `{{ var_auditd_disk_error_action }}`, will it insert a new line, or replace existing one?

Reply:
good call @yuumasato! fixed